src/index.css

```diff
@@ -195,3 +213,64 @@ hr {
     padding: 0 1.5em;
   }
 }
+
+/* Accordion */
+.accordion-section-container {
+  padding: 0 2em;
+}
+
+@media only screen and (max-width: 992px) {
+  .accordion-section-container {
+    padding: 0 1em;
+  }
+}
+
+.accordion-container {
+  color: var(--main-black);
+  overflow: hidden;
+  border-radius: 10px;
+}
+
+.accordion-header {
+  cursor: pointer;
+  font-family: HelveticaNeueMed;
+  font-size: 20px;
+  background-color: var(--main-cream);
+  padding: 1rem;
+}
+
+.accordion-header:hover {
+  background-color: var(--light-yellow);
+}
+
+.accordion-header-large {
+  font-size: 24px;
+  padding: 1.2rem;
+}
+
+.accordion-body {
+  font-size: 18px;
+  background-color: var(--main-grey);
+  color: var(--white);
+  padding: 1.5rem 3rem;
+}
+
+.accordion-body-large {
+  padding: 2.5rem 4rem;
+}
+
+@media only screen and (max-width: 992px) {
+  .accordion-header {
+    font-size: 16px;
+    padding: 1rem;
+  }
+
+  .accordion-header-large {
+    font-size: 20px;
```

Comment:
24px might look better? or if we switch to 20 at 786 instead of 992 , since rn both the header and the body font are 20px at this screen size which looks a bit weird